cfme/tests/configure/test_ntp_server.py

```diff
@@ -12,7 +11,8 @@
 
 
 pytestmark = [test_requirements.configuration]
-
+ntp_servers_keys = ['ntp_server_1', 'ntp_server_2', 'ntp_server_3']
```

Comment:
**OPTIONAL:** Is this one of the only places we could pull it from the ```_ntp_servers``` attr on the ```ServerInformation``` class

Reply:
added a property for that